### PR TITLE
Add common image attributes

### LINKED ISSUE #525: Allow alt attribute on img by default
## To Reproduce

Step by step instructions to reproduce the behavior:

1. Sanitize `` with `alt` attribute

## Expected behavior

`alt` attribute is retained.

## Describe the bug

`alt` attribute is removed.

## Details

In general the `alt` attribute is required by the HTML standard and should not be removed by default.

### PR DESCRIPTION
**Pull request by zadeviggers**
## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*

Allow common image attributes by default. If the `src` attribute is going to be enabled without images being enabled, it makes sense to also allow other commonly used tags to reduce confusion when using the library.

The `alt` attribute is also required on `` tags by the HTML standard so it shouldn't be removed by default.

Closes #525

## What kind of change does this PR introduce?
*(Check at least one)*

- [ ] Bug fix
- [ ] New feature
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [x] Other

## Make sure the PR fulfills these requirements:

- [x] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [x] The changelog is updated
- [x] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -780,10 +780,10 @@ sanitizeHtml.defaults = {
   disallowedTagsMode: 'discard',
   allowedAttributes: {
     a: [ 'href', 'name', 'target' ],
-    // We don't currently allow img itself by default, but this
+    // We don't currently allow img itself by default, but these
     // would make sense if we did. You could add srcset here,
     // and if you do the URL is checked for safety
-    img: [ 'src' ]
+    img: [ 'src', 'alt', 'title', 'width', 'height', 'loading' ]
```
Comment (boutell) [inline index.js:786]: This is reasonable — how about adding `srcset` while you're at it?

Comment (boutell) [reply index.js:786]: Also be sure to update the changelog.

**Review by boutell — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (zadeviggers) [reply index.js:786]: Okay

**Review by boutell — APPROVED** [review]
Thanks!

**Pull request merged**